scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/JobStatus.java

```diff
@@ -99,6 +99,38 @@
         definition = def;
         this.jobAlive = jobAlive;
     }
+    
+    public static JobStatus findPriority(String name) {
+        if (name.equalsIgnoreCase(PENDING.toString())) {
```

Comment:
I think the implementation of this class could be replaced by a simple for loop since it is possible to iterate over all constants with method `values()` inside the enum. This way it would be much more maintainable.